src/syscheckd/include/syscheck.h

```diff
@@ -41,6 +41,7 @@
 /* Global config */
 extern syscheck_config syscheck;
 extern int sys_debug_level;
+extern int audit_queue_full_reported;
```

Comment:
I have been analyzing this issue in depth. But I think that due to the simplicity of the operation of this variable, it is not necessary to use atomic operations (we can avoid even performance extras).
The only potential danger is not to generate a warning related to the queue full of audit, it would only be necessary to wait for the next scan to see it again.
I have also made a test with helgrind, and it has not generated any problem related to this variable.

[helgrind_test.log](https://github.com/wazuh/wazuh/files/10943992/helgrind_test.log)